### PR TITLE
Bump up default font sizes, and fix Download file button

### LINKED ISSUE #9078: Increase base font size on Add-Ons pages
### Describe the problem and steps to reproduce it:
The font size across the app should be bumped up. It's not accessible to those with even the slightest visual impairments.

### What happened?
I've been having some issues with my vision and was searching for a Firefox extension that would help make text more readable.

I found the following extension, which is great but it's ironic that the font size on the "About this Extension" section is unreadable in my current state.

### What did you expect to happen?
I expected I would be able to read about the extension and search for different extensions on your site.

### Anything else we should know?
Extension details page: https://addons.mozilla.org/en-US/firefox/addon/readability-based-reader-view/

Search results page:  https://addons.mozilla.org/en-US/firefox/search/?platform=mac&q=readability

image

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #9078 

It looks like people are happy with this change, so requesting a review. Will wait for @verdi's ok before landing.

### CODE REVIEW

File: src/amo/css/inc/vars.scss [hunk 1]
```diff
@@ -23,14 +23,13 @@ $default-font-family: 'Clear Sans', Helvetica, Arial, sans-serif;
 
 // Font-Size Variables
 $default-line-height: 19px;
-$font-size-default: 14px;
+$font-size-default: 16px;
 $font-size-micro: 10px;
```
Comment (eviljeff) [inline src/amo/css/inc/vars.scss:27]: not sure where we use it, but should `micro` increase to `12px`?

File: src/amo/components/InstallButtonWrapper/styles.scss [hunk 2]
```diff
@@ -15,6 +15,6 @@
   a:link {
     color: $sub-text-color;
     font-size: $font-size-s;
-    font-weight: lighter;
+    font-weight: 300;
```
Comment (eviljeff) [inline src/amo/components/InstallButtonWrapper/styles.scss:18]: just pointing out we're changing a relative `font-weight` to an absolute here.  (And that 300 is the correct weight from whatever style guide we're adhering to)

File: src/amo/css/inc/vars.scss [hunk 3]
```diff
@@ -23,14 +23,13 @@ $default-font-family: 'Clear Sans', Helvetica, Arial, sans-serif;
 
 // Font-Size Variables
 $default-line-height: 19px;
```
Comment (eviljeff) [inline src/amo/css/inc/vars.scss:25]: might cause unintended consequences but generally we'd increase the default line-height if the default font size increased?

**Review by eviljeff — APPROVED** [review]
definite 👍  to no more `m-smaller`

File: src/amo/css/inc/vars.scss (hunk 1 above)
Comment (bobsilverberg) [reply src/amo/css/inc/vars.scss:27]: Good idea. I'll check, rename it to `$font-size-xs`, and update it to `12px`.

File: src/amo/css/inc/vars.scss (hunk 3 above)
Comment (bobsilverberg) [reply src/amo/css/inc/vars.scss:25]: That reminds me that the style guide has specific line heights for each size. I'll make sure this one is decent, and then maybe we should have a follow-up to update them all. It's not as straightforward as this patch as I don't think we have standard fonts defined with multiple properties, but we probably should.

File: src/amo/components/InstallButtonWrapper/styles.scss (hunk 2 above)
Comment (bobsilverberg) [reply src/amo/components/InstallButtonWrapper/styles.scss:18]: I don't see any specific font-weights in the design guide that would apply to this. I just chose one that is light, but not so light that it becomes impossible to see the link.

File: src/amo/css/inc/vars.scss (hunk 3 above)
Comment (bobsilverberg) [reply src/amo/css/inc/vars.scss:25]: I changed this from `19px` to `24px`, which is what is suggested for Body MD in the design guide. I am going to file two follow-ups as well, one for reducing the number of font sizes and one for using suggested line heights along with the new font sizes.

**Pull request merged**